src/components/profile/edit/EditIdentities.vue

```diff
@@ -9,7 +9,14 @@
     formName="Edit identities"
   >
     <header class="profile__section-header" ref="header" tabindex="-1">
-      <h2>Identities</h2>
+      <div class="profile__section-header__title-info">
```

Comment:
In my _we don't need divs_ mode: if we change the `div` to an `h2` and put the _title_ directly in that. We just need to change the display of the `Tooltip` to `inline-flex` and it would look that same I guess.

Reply:
I originally tried without the div, but the css got very messy - the header already has flex set which needs to be overcome if the tooltip is outside the <h2>, and if it's inside then we need to unset all the h2 css on the tooltip.